* remove check for tclInt.h and tkInt.h in source folder, not necessary
author: Miguel Dias Costa (National University of Singapore)
--- configure.orig	2008-04-18 15:21:26.000000000 +0800
+++ configure	2020-01-17 12:44:04.311685907 +0800
@@ -7217,12 +7217,12 @@
 	        TCL_INCLUDES="${TCL_INCLUDES} ${TCL_INCLUDE_SPEC} `echo "${TCL_INCLUDE_SPEC}" | sed -e 's/Headers/PrivateHeaders/'`"; fi
 	        ;;
 	esac
-    else
-	if test ! -f "${TCL_SRC_DIR}/generic/tclInt.h" ; then
-	    { { echo "$as_me:$LINENO: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}" >&5
-echo "$as_me: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}" >&2;}
-   { (exit 1); exit 1; }; }
-	fi
+    #else
+	#if test ! -f "${TCL_SRC_DIR}/generic/tclInt.h" ; then
+	#    { { echo "$as_me:$LINENO: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}" >&5
+#echo "$as_me: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}" >&2;}
+   #{ (exit 1); exit 1; }; }
+	#fi
     fi
 
 
@@ -7278,12 +7278,12 @@
 	        TK_INCLUDES="-I\"${TK_BIN_DIR}/Headers\" -I\"${TK_BIN_DIR}/PrivateHeaders\" ${TK_INCLUDES}"; fi
 	        ;;
 	esac
-    else
-	if test ! -f "${TK_SRC_DIR}/generic/tkInt.h" ; then
-	    { { echo "$as_me:$LINENO: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}" >&5
-echo "$as_me: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}" >&2;}
-   { (exit 1); exit 1; }; }
-	fi
+    #else
+#	if test ! -f "${TK_SRC_DIR}/generic/tkInt.h" ; then
+#	    { { echo "$as_me:$LINENO: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}" >&5
+#echo "$as_me: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}" >&2;}
+#   { (exit 1); exit 1; }; }
+#	fi
     fi
 
 
